### ISSUE TITLE
Bradford says "eliminate the remaining hostiles in the area" when the actual next objective is to evac

### ISSUE THREAD

**Issue by Iridar** [long-war-2/lwotc#1512]
Had this happen on a "recover item from advent cache" mission.

**Comment by Iridar** [comment]
See the objective display:

https://i.imgur.com/lw2FhMe.jpg

**Comment by pledbrook** [comment]
Yes. You'll get this a lot on LWOTC missions because the voice lines are vanilla, and most of its missions are "kill all the enemies", unlike with LWOTC. I don't know what we can reasonably do.

**Comment by pledbrook** [comment]
Note that this was exactly the same in Long War 2.

**Comment by Iridar** [comment]
Perhaps you could kill that particular narrative line, so that Bradford just says nothing in this case. Though if that would require dealing with kismet, it's probably not worth it.
